src/plugins/catalog/catalog.yml

```diff
@@ -18,29 +18,29 @@ catalog:
     en: Hover effect
     de: Hover-Effekt
   hoverEffect_none:
-    en: No effect
+    en: no effect
     de: Kein Effekt
   hoverEffect_colorizeZoom:
-    en: Color and zoom
+    en: color and zoom
```

Comment:
I thought we were going in the direction to have all control texts in sentence case (also in English), and I remember Radu criticizing some that weren't, so I'm not sure about these two changes :thinking:

Reply:
Now we're consistent in English with all radios (control content) as lowercased and all CTAs and labels capitalized.

Reply:
I see, thanks!